Разметка всех страниц

NadezhdaObsholova opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/1664379/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/1664379-device-36/2/

NadezhdaObsholova committed:
Update index.html

разметка шапки

NadezhdaObsholova committed:
добавление хиро-блока

добавление хиро-блока

NadezhdaObsholova committed:
категории товаров

добавлен блок с категориями товаров

NadezhdaObsholova committed:
добавление блока услуг

NadezhdaObsholova committed:
добавлена секция "о нас"

NadezhdaObsholova committed:
добавлен блок подписки

NadezhdaObsholova committed:
добавлен подвал

NadezhdaObsholova committed:
исправление ошибок после валидатора

smart-grizzly commented on index.html:
```diff
@@ -1,12 +1,314 @@
 <!DOCTYPE html>
 <html lang="ru">
+
```
https://up.htmlacademy.ru/htmlcss/36/criteries#HTML-07
Ошибки в валидаторе типа `error` - их надо исправить)

NadezhdaObsholova replied:
валидатор показывает только ошибки типа "нет пути картинки", пустой атрибут action в теге form. я пока не могу их исправить, так как графика будет позже.
а что, кстати, писать в action? куда должны отправляться данные?

smart-grizzly replied:
https://up.htmlacademy.ru/htmlcss/36/criteries#HTML-04

smart-grizzly commented on index.html:
```diff
@@ -1,12 +1,314 @@
+      <a class="header-logo"></a>
+
+      <form class="header-search" action="" method="get">
+        <input name="search" placeholder="Поиск по сайту" type="search">
```
https://up.htmlacademy.ru/htmlcss/36/criteries#HTML-05
нужен свой `label` - для всех инпутов 👍

smart-grizzly commented on index.html:
```diff
@@ -1,12 +1,314 @@
+        <a class="catalog" href="catalog.html">Каталог товаров</a>
+        <button type="button"></button>
+        <ul class="catalog-list visually-hidden">
+          <li class="catalot-item">Виртуальная реальность</li>
```
1) `catalot` = > `catalog`
Ошибка английского слова
2) и тут ошибка. - это ссылка и внутри картинка! Потому что это интерактивный элемент, а не просто текст

smart-grizzly commented on index.html:
```diff
@@ -1,12 +1,314 @@
+          </div>
+          <a class="open-good-page" href="">Подробнее</a>
+          <table class="good-features">
+            <tr>
```
все тэги должны иметь классы - так как в дальнейшем будем стилизовать только по классам 👍 
Исправь поя-ста везде по проекту!

smart-grizzly commented on index.html:
```diff
@@ -1,12 +1,314 @@
+        <h2 class="visually-hidden">Категории товаров</h2>
+        <ul class="goods-categories-list">
+          <li class="goods-categories-item">
+            <img src="" class="good-icon">
```
1) для тэга `img` атрибут `alt` обязателен и это нужно везде исправить 👍

NadezhdaObsholova replied:
в данном случае изображение является декоративной иконкой, не несущей смысловой нагруки. оставить alt пустым?

smart-grizzly replied:
да верно!

smart-grizzly commented on index.html:
```diff
@@ -1,12 +1,314 @@
+        </div>
+
+        <form class="subscribe-form" action="" method="get">
+          <input id="email" placeholder="Ваш e-mail" type="email">
```
тут тоже нужен свой связной `label`

smart-grizzly commented on index.html:
```diff
@@ -1,12 +1,314 @@
+
+      <div>Тел.: +7 (812) 812-12-12</div>
+
+      <ul class="sotial-list">
```
`sotial` => `social`

smart-grizzly commented on index.html:
```diff
@@ -1,12 +1,314 @@
+      </ul>
+
+
+      <a class="html-academi-logo" href="https://htmlacademy.ru/">
```
`academi` => `academy`

smart-grizzly commented on index.html:
```diff
@@ -1,12 +1,314 @@
+        </ul>
+      </nav>
+
+      <div>Тел.: +7 (812) 812-12-12</div>
```
https://htmlacademy.ru/blog/html/mailto
изучи как делать телефон интерактивным 👍

smart-grizzly commented on index.html:
```diff
@@ -1,12 +1,314 @@
+          </table>
+        </div>
+
+
```
и еще смотри я так понимаю у тебя между блоками 1 пустой строке,  а тут их две(
Надо придерживаться единообразия 👍

smart-grizzly commented on index.html:
```diff
@@ -1,12 +1,314 @@
+          </tr>
+        </table>
+
+
```
тут тоже 2 пустые строки(

NadezhdaObsholova committed:
исправление замечаний

не исправлены формы и интерактивный телефон

NadezhdaObsholova committed:
исправлены замечания для главной

исправлены указанные замечания (интерактивный телефон, лейбл для инпута в формах) добавлены ариа-лейбл для кнопок без текста.
добавлен поп-ап.

NadezhdaObsholova committed:
разметка страницы каталога

Добавлена разметка каталога, добавлен поп-ап корзины. исправлены наименования классов.

NadezhdaObsholova committed:
исправление ошибок после валидатора

NadezhdaObsholova committed:
замена таблиц списком dl

NadezhdaObsholova committed:
вернула таблицы

прочитала о полезности таблиц и решила вернуть их обратно
